scripts/generate_swagger_spec.sh

```diff
@@ -19,3 +19,5 @@ go install github.com/swaggo/swag/cmd/swag
 
 echo "+ Generate swagger spec"
 swag init -g cmd/tidb-dashboard/main.go
+go mod vendor
+swag init --parseVendor -g cmd/tidb-dashboard/main.go
```

Comment:
In issue swaggo/swag#567, We cannot parse struct correctly in external packages without `--parseVendor` flag.
So we change the commands here:
1. Swag generates `docs/docs.go` file, but without the JSON field related to external packages.
2. Enable `go vendor` for this package
3. Swag regenerates `docs/docs.go` file.

Reply:
Can we remove line 21?

Reply:
No, it will cause the 3rd step panic, this panic is a bug inner swaggo.